documentation/staging/content/userguide/managing-domains/accessing-the-domain/port-forward.md

```diff
@@ -63,19 +63,52 @@ This behavior depends on your version and domain resource configuration:
   [schema](https://github.com/oracle/weblogic-kubernetes-operator/blob/main/documentation/domains/Domain.md).
 
 * For Istio-enabled domains running Istio versions prior to 1.10,
-  the operator already adds a
-  network channel with a `localhost` listen address for each
-  existing port. This means that no additional configuration is required
-  to enable port forwarding when Istio is enabled.
+  you must add an additional network channel to the WebLogic Administration Server 
```

Comment:
Is the additional network channel still needed if user only wants to access console (and not WLST)? I think WLST use-case is not very common and we may want to mention about the additional channel only when WLST access is required.

Reply:
Technically, it's not needed if user only wants to access using console.  We can update the description to include if online WLST access is required.  It is becoming very confusing with all these restrictions so I omitted it as the additional network channel works for both console and online WLST.